Directory.Build.props

```diff
@@ -7,8 +7,22 @@
 
     <!-- This is used to disable some build properties. -->
     <IsExamplesSolution Condition="'$(SolutionName)' == 'Examples' ">true</IsExamplesSolution>
+
+    <!-- Remove below entry before merge, using it to build and test Redfield -->
+	<Redfield>true</Redfield>
```

Comment:
TODO: don't forget to remove this before merge :)

Reply:
All tests with `Redfield=true` ran success. Remove this entry, will rely on our pipeline to introduce flag.